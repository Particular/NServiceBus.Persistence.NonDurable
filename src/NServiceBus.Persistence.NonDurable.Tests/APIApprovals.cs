namespace NServiceBus.Persistence.NonDurable.Tests
{
    using NServiceBus;
    using NUnit.Framework;
    using Particular.Approvals;
    using PublicApiGenerator;

    [TestFixture]
    public class APIApprovals
    {
        [Test]
        public void ApproveNonDurablePersistence()
        {
            var publicApi = typeof(NonDurablePersistence).Assembly.GeneratePublicApi(new ApiGeneratorOptions { ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" } });
            Approver.Verify(publicApi);
        }
    }
}