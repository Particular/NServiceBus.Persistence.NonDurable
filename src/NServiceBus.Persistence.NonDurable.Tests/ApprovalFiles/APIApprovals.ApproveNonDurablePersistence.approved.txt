[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute(@"NServiceBus.Persistence.NonDurable.PersistenceTests, PublicKey=00240000048000009400000006020000002400005253413100040000010001005530b5fb485f7355613e8f40ff82c4b3a2cf1e0fbd73f0600c0696c2f015dd2cc2a1bb1697361b207e3d22efb4304f5aaf45d10fe2577ff1c16c9ba03a5264d1753feee9d042ab2b01eab864118aadbc4085cb0c8917f5ea004185d8f915b3b73e298f338c9609b28c8680ee08c4b99427e0282009f800d4901f76c19d0b57c4")]
[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute(@"NServiceBus.Persistence.NonDurable.Tests, PublicKey=00240000048000009400000006020000002400005253413100040000010001005530b5fb485f7355613e8f40ff82c4b3a2cf1e0fbd73f0600c0696c2f015dd2cc2a1bb1697361b207e3d22efb4304f5aaf45d10fe2577ff1c16c9ba03a5264d1753feee9d042ab2b01eab864118aadbc4085cb0c8917f5ea004185d8f915b3b73e298f338c9609b28c8680ee08c4b99427e0282009f800d4901f76c19d0b57c4")]
namespace NServiceBus
{
    public class static NonDurableOutboxSettingsExtensions
    {
        public static NServiceBus.Outbox.OutboxSettings TimeToKeepDeduplicationData(this NServiceBus.Outbox.OutboxSettings settings, System.TimeSpan time) { }
    }
    public class NonDurablePersistence : NServiceBus.Persistence.PersistenceDefinition { }
}